src/docs/integration/linux/overview.md

```diff
@@ -0,0 +1,78 @@
+
+### Runtime Structure
+
+#### Partition Structure
+
+The following partition structure is a loose guide for how the system is organizaed. The partition structure is completely customizable, the following structure opinions were chosen as a general base line for producing fault tolerant embedded systems.
+
+* U-Boot Environment
+* Boot Part A/B Updates
+* RootFS A/B Updates (Squashfs read-only)
+* Overlay FS Data Partition (read write)
+
+The U-Boot Environment is used as a persistent data store for peridiod to track variables such as a boot counter, a validation flag, which partition slot is active, the contents of each slot, etc. These variables can be used at early boot time to perform roll back logic, for example, the validation flag can be set after the program reaches the desired runtime state after an update. If has not been set, something likely went wrong and the hardware watchdog reset the board. Fpr example:
+
+```text
+peridio_revert=\
+    if test ${peridio_active} = "a"; then\
+        echo "Reverting to partition B";\
+        setenv peridio_active "b";\
+    else\
+        echo "Reverting to partition A";\
+        setenv peridio_active "a";\
+    fi
+
+peridio_init=\
+    if test ${peridio_booted} = 1; then\
+        if test ${peridio_validated} = 0; then\
+            run peridio_revert;\
+            setenv peridio_validated 1;\
+            saveenv;\
+        fi;\
+    fi;\
+    if test ${peridio_active} = "a"; then\
+        setenv bootpart 2;\
+        setenv rootdevpath ${a.peridio_rootfs_part_devpath};\
+        setenv datadevpath ${a.peridio_datafs_part_devpath};\
+    else\
+        setenv bootpart 3;\
+        setenv rootdevpath ${b.peridio_rootfs_part_devpath};\
+        setenv datadevpath ${b.peridio_datafs_part_devpath};\
+    fi;
+
+bootcmd=run peridio_init my_boot_command
+```
+
+### Peridio Cloud Requirements
+
+Building and connecting a reference design system to Peridio requires some additional configuration in your Peridio Cloud organization. The following steps wll prepare your Peridio Cloud organization to connect devices running the default reference platform.
+
+* Reference designs use the product name `avocado` by default, [Create a new product](/platform/guides/creating-products) named `avocado` in your Peridio Cloud organization.
+* [Generate a Certificate Authority](/platform/guides/creating-x509-certificates-with-openssl) for signing trusted device identities.
+* [Configure Peridio Cloud to trust your Certificate Authority](/platform/guides/creating-ca-certificates)
```

Comment:
The guide linked to in `/platform/guides/creating-ca-certificates` already within itself links to `/platform/guides/creating-x509-certificates-with-openssl`.  It may be possible to simplify these two bullets into one like:

- Configure Peridio Cloud to trust your Certificate Authority that signs device identities. For more information, see  [creating CA certificates](/platform/guides/creating-ca-certificates).

Reply:
I might push back here, one is about making the certificates, the other is about registering them with Peridio Cloud. While there are links inside the x509 guide to CA certs in Peridio Cloud reference, it's not as clear that there are two distinct steps required.

Reply:
I'm cool with that read - up to you.